### PR TITLE
.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by Toytoles** [comment]
> @microsoft-github-policy-service agree [company="{hctec}"]

**Comment by Toytoles** [comment]
> @microsoft-github-policy-service agree [company="{hctec}"]

**Comment by Toytoles** [comment]
@microsoft-github-policy-service agree [company="{your company}"]

**Comment by Toytoles** [comment]
@microsoft-github-policy-service agree company=hctec

**Comment by Toytoles** [comment]
@microsoft-github-policy-service agree

File: manifests/h/hctec/anydesk/7.0.14/hctec.anydesk.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: hctec.anydesk
```
Comment (Trenly) [inline manifests/h/hctec/anydesk/7.0.14/hctec.anydesk.installer.yaml:4]:
```suggestion
PackageIdentifier: AnyDeskSoftwareGmbH.AnyDesk.HCTecSupport
```

File: manifests/h/hctec/anydesk/7.0.14/hctec.anydesk.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: hctec.anydesk
```
Comment (Trenly) [inline manifests/h/hctec/anydesk/7.0.14/hctec.anydesk.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: AnyDeskSoftwareGmbH.AnyDesk.HCTecSupport
```

File: manifests/h/hctec/anydesk/7.0.14/hctec.anydesk.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: hctec.anydesk
```
Comment (Trenly) [inline manifests/h/hctec/anydesk/7.0.14/hctec.anydesk.yaml:4]:
```suggestion
PackageIdentifier: AnyDeskSoftwareGmbH.AnyDesk.HCTecSupport
```

**Review by Trenly — CHANGES_REQUESTED** [review]
Anydesk is already available at AnyDeskSoftwareGmbH.AnyDesk. This PR has many deficiencies and a new one should be opened with accurate metadata if this is a new package to be merged

**Comment by Trenly** [comment]
Close with reason: New PR should be created rather than trying to fix this one;

**Pull request closed without merging**